client/components/domains/domain-registration-suggestion/index.jsx

```diff
@@ -319,8 +319,9 @@ class DomainRegistrationSuggestion extends React.Component {
 		let buttonClasses, buttonContent;
 
 		if ( domain ) {
-			const testTLDs = [ '.de' ];
-			// Grab everything after the first dot, so 'example.co.uk' will
+			const testTLDs = [ '.de', '.in', ];
```

Comment:
Small nitpick, but dangling commas in this format are discouraged. That's because it's easy to make this mistake: `[ '.de', '.in',, ]`. Now you have three elements, one empty.
We even had an eslint rule added for it ([`comma-dangle`](https://eslint.org/docs/rules/comma-dangle)), but it was reverted (see p4TIVU-889-p2) along with eslint v4 changes: https://github.com/Automattic/wp-calypso/pull/18108. Still, sooner or later, we'll re-introduce it :)
Changing this line to multiline format should be the best way to go (as it should have been from the beginning - of course, not your fault):
```
const testTLDs = [
    '.de',
    '.in',
];
```